zstacklib/zstacklib/gpu/vendors/vastai.py

```
@@ -0,0 +1,75 @@
+        for elem in mem_data["elem"]:
+            pci_address = cls.normalize_pci_address(elem.get("pci_bus", "N/A"))
+            serial = elem.get("sn", "N/A")
+            
+            vals = elem.get("vals", {})
+            mem_val = vals.get("Physical", {}).get("value") or vals.get("Physical memory", {}).get("value", "N/A")
+            
+            memory = mem_val
+            if mem_val != "N/A":
+                try:
+                    memory = str(sizeunit.get_size(mem_val)) + "B"
+                except:
```

Ruff (0.14.13)

Do not use bare `except`

(E722)